central/metrics/central.go

```diff
@@ -277,6 +278,20 @@ var (
 		Name:      "msg_to_sensor_not_sent_count",
 		Help:      "Total messages not sent to Sensor due to errors or other reasons",
 	}, []string{"ClusterID", "type", "reason"})
+
+	bulkProcessBaselineCallCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I think this metric is redundant with metrics already in master and should be removed.

Reply:
@dashrews78 What do you think?

Reply:
Which metric already collected makes it redundant?  This illustrates why this should have been follow-on in a separate PR.

Reply:
Should I move the changes to the metrics to another PR?

Reply:
I have moved the metrics to another branch. I will soon create a ticket and push the branch with the metrics.